lib/compilers/argument-parsers.ts

```diff
@@ -339,12 +323,10 @@ export class ClangParser extends BaseParser {
         try {
             const options = await this.getOptions(compiler, this.getMainHelpOptions().join(' '));
 
-            const filename = this.getExampleFilepath();
-
             this.mllvmOptions = new Set(
-                _.keys(await this.getOptions(compiler, this.getHiddenHelpOptions(filename).join(' '), false, true)),
+                _.keys(await this.getOptions(compiler, this.getHiddenHelpOptions().join(' '), false, true)),
             );
-            this.setCompilerSettingsFromOptions(compiler, options);
+            await this.setCompilerSettingsFromOptions(compiler, options);
```

Comment:
@partouf here - my only guess is... if somehow this took ages and we "discovered" and wrote the output before this completed (without the await)... but... seems unlikely

Reply:
issue found, updated the description and checked in a "real fix". testing in staging...